flipkart/index.html

```diff
@@ -0,0 +1,711 @@
+    </div>
+    <div class="display">
+      <p>16.0cm (6.3 inch) Full HD+Display </p>
+      <p>48MP + 5MP</p>
+    </div>
+    <div class="battery">
+      <p>4000 mah</p>
+      <p>13 MP Front Camera</p>
+    </div>
+  </div>
+  <br>
+  <hr>
+
+  <div class="mobile-1stmobile">
+    <div class="mobile-note7">
+      <div class="mobile-note7-image">
+        <img src="img/note7.jpeg" alt="">
+      </div>
+      <div class="mobile-note8-specs">
+        <h4>Redmi Note 8 (Moonlight White, 128 GB)</h4>
+        <p>#justhere</p>
+        <button class="button-rating">
+          <a href="">4.1★</a>
+        </button>
+        <p>2211 Reviews</p>
+        <img src="img/assured.png" alt="">
+        <h4>35,999</h4>
+        <p>12,229 22% off <br>No Cost EMI <br>Upto ₹13,250 off on Exchange </p>
+      </div>
+    </div>
+  </div>
+  <div class="note8-ram">
+    <div class="ram">
+      <p>6 GB | 128 GB ROM</p>
+    </div>
+    <div class="display">
+      <p>16.0cm (6.3 inch) Full HD+Display </p>
+      <p>48MP + 5MP</p>
+    </div>
+    <div class="battery">
+      <p>4000 mah</p>
+      <p>13 MP Front Camera</p>
+    </div>
+  </div>
+  <br>
+  <hr>
+
+  <div class="mobile-1stmobile">
+    <div class="mobile-note7">
+      <div class="mobile-note7-image">
+        <img src="img/rn8.jpeg" alt="">
+      </div>
+      <div class="mobile-note8-specs">
+        <h4>Mi A3 (More Than White, 128 GB)</h4>
+        <p>#justhere</p>
+        <button class="button-rating">
+          <a href="">4.0★</a>
+        </button>
+        <p>211 Reviews</p>
+        <img src="img/assured.png" alt="">
+        <h4>22,288</h4>
+        <p>10,229 22% off <br>No Cost EMI <br>Upto ₹22,250 off on Exchange </p>
+      </div>
+    </div>
+  </div>
+  <div class="note8-ram">
+    <div class="ram">
+      <p>6 GB | 128 GB ROM</p>
+    </div>
+    <div class="display">
+      <p>16.0cm (6.3 inch) Full HD+Display </p>
+      <p>48MP + 5MP</p>
+    </div>
+    <div class="battery">
+      <p>4000 mah</p>
+      <p>13 MP Front Camera</p>
+    </div>
+  </div>
+  <br>
+  <hr>
+
+  <div class="mobile-1stmobile">
+    <div class="mobile-note7">
+      <div class="mobile-note7-image">
+        <img src="img/image4.PNG" alt="">
+      </div>
+      <div class="mobile-note8-specs">
+        <h4>Redmi K20 (Flame Red, 128 GB)</h4>
+        <p>#justhere</p>
+        <button class="button-rating">
+          <a href="">4.4★</a>
+        </button>
+        <p>711 Reviews</p>
+        <img src="img/assured.png" alt="">
+        <h4>32,288</h4>
+        <p>9,229 22% off <br>No Cost EMI <br>Upto ₹12,250 off on Exchange </p>
+      </div>
+    </div>
+  </div>
+  <div class="note8-ram">
+    <div class="ram">
+      <p>6 GB | 128 GB ROM</p>
+    </div>
+    <div class="display">
+      <p>16.0cm (6.3 inch) Full HD+Display </p>
+      <p>48MP + 5MP</p>
+    </div>
+    <div class="battery">
+      <p>4000 mah</p>
+      <p>13 MP Front Camera</p>
+    </div>
+  </div>
+  <br>
+  <footer>
```

Comment:
Actually, if the footer looks the same on mobile and web view the ideal way is to write one code block and you can write a media query for the mobile view.